### PR TITLE
[migrations, 0x.js, contract-wrappers, fill-scenarios, order-utils, order-watcher] remove v2 contract artifacts

### PR DESCRIPTION
**Pull request by feuGeneA**
## Description

`0x-monorepo$ git rm -rf packages/migrations/artifacts/2.0.0`
And then removed the (unsurprisingly numerous) references to those artifacts.

## Testing instructions

`0x-monorepo$ yarn build && yarn test`

## Types of changes

* Bug fix (non-breaking change which fixes an issue)
    * Bug/issue in the sense of the annoyance of always having `yarn build` update those artifacts and then having `git` flag them as changed in the working checkout.

## Checklist:

*   [x] Prefix PR title with `[WIP]` if necessary.
*   [x] Prefix PR title with bracketed package name(s) corresponding to the changed package(s). For example: `[sol-cov] Fixed bug`.
*   [x] Add tests to cover changes as needed.
*   [x] Update documentation as needed.
*   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

**Comment by feuGeneA** [comment]
Current CI failure:
```
@0xproject/order-utils
 | packages/order-utils/test/exchange_transfer_simulator_test.ts:10:41 - error TS2307: Cannot find module '../src/generated_contract_wrappers/dummy_erc20_token'.
 | 
 | 10 import { DummyERC20TokenContract } from '../src/generated_contract_wrappers/dummy_erc20_token';
 |                                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 | 
 | error An unexpected error occurred: "Command failed.
 | Exit code: 2
 | Command: sh
 | Arguments: -c tsc
 | Directory: /home/circleci/repo/packages/order-utils
 | Output:
 | ".
 | info If you think this is a bug, please open a bug report with the information provided in "/packages/order-utils/home/circleci/repo/packages/order-utils/yarn-error.log".
 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
 | ERROR: "transpile" exited with 1.
 | error An unexpected error occurred: "Command failed.
 | Exit code: 1
 | Command: sh
 | Arguments: -c run-s pre_build transpile copy_monorepo_scripts
 | Directory: /home/circleci/repo/packages/order-utils
 | Output:
 | ".
 | info If you think this is a bug, please open a bug report with the information provided in "/packages/order-utils/home/circleci/repo/packages/order-utils/yarn-error.log".
 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
`yarn build` failed with exit code 1
```

File: packages/migrations/package.json [hunk 1]
```diff
@@ -11,29 +11,29 @@
...
         "compile:v1": "sol-compiler --artifacts-dir artifacts/1.0.0 --contracts Exchange_v1,DummyERC20Token,ZRXToken,WETH9,TokenTransferProxy_v1,MultiSigWallet,MultiSigWalletWithTimeLock,MultiSigWalletWithTimeLockExceptRemoveAuthorizedAddress,TokenRegistry",
-        "compile:v2": "sol-compiler --artifacts-dir artifacts/2.0.0 --contracts ERC20Token,DummyERC20Token,ERC721Token,DummyERC721Token,ERC20Proxy,ERC721Proxy,Exchange,Forwarder,MultiSigWalletWithTimeLockExceptRemoveAuthorizedAddress,ZRXToken,WETH9,IWallet,IValidator",
-        "compile:v2-beta-testnet": "sol-compiler --artifacts-dir artifacts/2.0.0-beta-testnet --contracts AssetProxyOwner,ERC20Proxy,ERC721Proxy,Exchange,Forwarder,IWallet,IValidator,ERC20Token,ERC721Token"
+        "compile:v2": "sol-compiler --artifacts-dir artifacts/2.0.0 --contracts AssetProxyOwner,ERC20Token,DummyERC20Token,ERC721Token,DummyERC721Token,ERC20Proxy,ERC721Proxy,Exchange,Forwarder,MultiSigWalletWithTimeLockExceptRemoveAuthorizedAddress,ZRXToken,WETH9,IWallet,IValidator",
```
[the remark this answers is not in the document]
Comment (feuGeneA) [reply packages/migrations/package.json]: We do.  Without it, `migrations$ yarn build` yields:

```
src/2.0.0/migration.ts:10:41 - error TS2307: Cannot find module './contract_wrappers/asset_proxy_owner'.

10 import { AssetProxyOwnerContract } from './contract_wrappers/asset_proxy_owner';
                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by fabioberger** [comment]
Disregard my comment, the Kovan addresses are already in there :)

**Comment by feuGeneA** [comment]
`test-publish` failure says:

```
$ { echo "test"; sleep 1; echo "test"; sleep 1; echo "test@example.com"; } | npm login --registry=http://localhost:4873 && IS_LOCAL_PUBLISH=true run-s script:publish test:installation:local
Username: test

Password: npm ERR! cb() never called!

npm ERR! This is an error with npm itself. Please report this error at:
npm ERR!     <https://github.com/npm/npm/issues>
```

I saw this before and asked Alex about it and [he said](https://0xproject.slack.com/archives/CALK56XT7/p1534443539000100) "I’ve seen this once before and I think it has to do with an incorrect timing with `npm login`.  I struggled to find a way to non-interactively login to npm. The current approach is just `{ echo "test"; sleep 1; echo "test"; sleep 1; echo "test@example.com"; } | npm login` but it looks like it might be a bit brittle."

**Pull request merged**